src/tests/ftest/control/dmg_pool_query_test.py

```diff
@@ -95,7 +95,9 @@ def test_pool_query_basic(self):
                     "objects", path="/run/exp_vals/rebuild/*"),
                 "records": self.params.get(
                     "records", path="/run/exp_vals/rebuild/*")
-            }
+            },
+            "enabled_ranks": None,
+            "disabled_ranks": None,
```

Comment:
The comma should be removed
```suggestion
            "disabled_ranks": None
```